src/Uno.UWP/Data/Pdf/PdfPageRenderOptions.Android.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Windows.Foundation;
+using Windows.UI;
+
+namespace Windows.Data.Pdf;
+
+public partial class PdfPageRenderOptions
+{
+	public Rect SourceRect { get; set; }
+
+	public bool IsIgnoringHighContrast { get; set; }
+
+	public uint DestinationWidth { get; set; }
+
+	public uint DestinationHeight { get; set; }
+
+	public Guid BitmapEncoderId { get; set; }
```

Comment:
Same comment here. If this property is unused, it should remain as NotImplemented